frontend/ui/network/manager.lua

```diff
@@ -105,23 +119,14 @@ function NetworkMgr:promptWifiOff(complete_callback)
 end
 
 function NetworkMgr:turnOnWifiAndWaitForConnection(callback)
-    NetworkMgr:turnOnWifi()
-    local timeout = 30
-    local retry_count = 0
-    local info = InfoMessage:new{ text = T(_("Enabling Wi-Fi. Waiting for Internet connection…\nTimeout %1 seconds."), timeout)}
+    local info = InfoMessage:new{ text = _("Enabling Wi-Fi. Waiting for Internet connection…") }
```

Comment:
For this one, I personnaly use:
```lua
        -- mieux si on utilise trapwidget, on peut ainsi continuer à lire pendant que ça connecte.
        -- local info = InfoMessage:new{
        local TrapWidget = require("ui/widget/trapwidget")
        local info = TrapWidget:new{
            text = "Connecting to friendly WIFI...",
            image_file = "../mykoreader/resources/info-wifi.png",
        }
```

Reply:
Where is the icon from? ;).

Reply:
Icon's besides the point, lol. :-P

(I mean, we can always consider it but we already have a whole series of 'em if desired.)

Reply:
(yep, just happen to be this: ![info-wifi](https://user-images.githubusercontent.com/24273478/88433423-50d8e380-cdfe-11ea-99ee-e9d788e51900.png) )